QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151022_002845	I welcome any swordsman with an aptitude.{nl}Let's defeat the demons together!
+QUEST_20151022_002846	You could say it were the demons that made it possible for us to reach a sub-master level at such a young age.{nl}I will repay them with my sword and shield.
+QUEST_20151022_002847	It's already been 4 years, but there are many people who still carry the scars of that day.{nl}In times like these, our cleric's blessing by the goddess must be shared.
+QUEST_20151022_002848	Huh, are you on a trip?{nl}Must be nice~ I can't move an inch with a mission here.
+QUEST_20151022_002849	To spread the blessing of the goddess, the it doesn't matter what kind of priest you are.{nl}
+QUEST_20151022_002850	Originally, I was following a different path, but I ended up entering the Krivis.{nl}I believe helping others through a conventional way seems most appropriate.
+QUEST_20151022_002851	For tired and sick people, we must become their shoulder rest.{nl}As such, priests must bear with heavy responsibilites.
+QUEST_20151022_002852	For the goddesses in absence, we must be the one to present everyone's hearts.
+QUEST_20151022_002853	Are you asking me to bless the path of the Priest?	Course, I'll gladily issue the blessing.{nl}I pray that your path will be brimming with the blessing of the goddess. 
+QUEST_20151022_002854	Anyone who shares the blessing of goddess has the right to be blessed by anyone.{nl}I pray that the goddess will watch over his future.
+QUEST_20151022_002855	You must be someone who has recently pursued the path of the Priest.{nl}If you don't mind my blessing, I will gradly bless you.
+QUEST_20151022_002856	It's a shame that people such as yourself has left the path of the Clerics half-way through.
+QUEST_20151022_002857	I was originally a Pardoner.{nl}If the path of the Priest end up not working out, consider the path of the Krivis.
+QUEST_20151022_002858	Of course, helping others through a conventional way is still great.{nl}However, make sure not to forget the choice of Bokor.
+QUEST_20151022_002859	The state of the world after the Medzio Diena has truly gotten much better, but the world is still in chaotic state. Worthwhile people's dedication is keenly needed.
+QUEST_20151022_002860	That enormous tree was the cataclysm. As well are the monsters. Acknowledges.{nl}However, the chaotic events occured after are the kingdom and the subjects to blame. If Medzio Diena were to occur at our hometown, this kind of situation would've never happened.
+QUEST_20151022_002861	The people such as yourself hoping to straighten out the chaos are interesting.{nl}It's a forte that only several amount of people carry in this kingdom.
+QUEST_20151022_002862	Bokors such as ourselves are the clergy that are meant to spread the blessing of goddess to the world.{nl}It's just that the our manner is bit unfamiliar.
```

Comment:
Bokors such as ourselves -> We, Bokors, are the ....
blessing of goddess -> blessing of the goddess